### PR TITLE
[#43] dashboard integration

### PR DESCRIPTION
**Pull request by dodoels**
- integrate with new endpoint
- populate data

# Dashboard
image

# Sidebar - No Progress
image

# Sidebar - < prediction 1
image

# Sidebar - Right after prediction 
image

# Sidebar - Between predictions
image

### CODE REVIEW

**Comment by arekmano** [comment]
UI 10/10

File: src/web_client/src/components/Sidebar.vue [hunk 1]
```diff
@@ -206,18 +159,36 @@ export default {
...
+                        this.report.curve.push({
+                            outcome: {
+                                mean: 1,
+                                std: 1,
+                                variance: 1
+                            }
+                        })
+                    }
+                    this.isPredicted = true
+                    console.log(this.report.curves)
```
Comment (arekmano) [inline src/web_client/src/components/Sidebar.vue]: debug code

File: src/web_client/src/components/Sidebar.vue [hunk 2]
```diff
@@ -206,18 +159,36 @@ export default {
...
+                },
+                response => {
+                    console.log('Connection Fail: get report')
```
Comment (arekmano) [inline src/web_client/src/components/Sidebar.vue]: debug code

**Review by arekmano — COMMENTED** [review]
looks good. only few console.logs

File: src/web_client/src/components/Sidebar.vue (hunk 2 above)
Comment (dodoels) [reply src/web_client/src/components/Sidebar.vue]: we don't have error handling work flow yet, so this debug code is there to monitor the app. We will need to replace all the error handling log to something else.

**Comment by TaimoorRana** [comment]
This looks awesome Bruce, GJ!

**Review by TaimoorRana — APPROVED** [review]

**Review by Vynny — APPROVED** [review]
Looks sharp!

**Review by Samuel-Campbell — APPROVED** [review]
looks amazing

**Pull request merged**